### ISSUE TITLE
Possible to disable or hide the bottom status line?

### ISSUE THREAD

**Issue by klundry** [sqshq/sampler#100]
Is there a way to hide or disable the status menu line at the bottom?

**Comment by daniejstriata** [comment]
Agree. I don't use the bottom bar. That would be one extra line to use! :-) We could even drop the line between the box heading and its contents. Maybe make a "compressed" textbox type.
![image](https://user-images.githubusercontent.com/13314239/107944570-fe783380-6f96-11eb-9c70-c9b7faa8fd1c.png)
